docs/DeveloperGuide.md

```diff
@@ -271,33 +271,95 @@ This feature has been enhanced based on the initial implementation of the `edit`
 
-The proposed edit mechanism is facilitated by `EditCommand`. It is supported by `EditCommandParser` where it will retrieve the attributes that can be edited.
+The proposed edit mechanism is facilitated by `EditCommand`. It is supported by `EditCommandParser` and `EditPersonDescriptor` 
+where it will retrieve the attributes that can be edited.
 
 The user is able to edit key attributes of the Candidate, such as
 * Phone Number
 * ApplicationStatus
-* InterviewStatus
 
 Based on the fields that are edited by the user, the EditCommandParser will retreive the information, and update the attributes accordingly.
-There are also some attributes that are dependent on another attribute, and we will introduce `Triggers` to update the attributes automatically.
-One of such is the `InterviewStatus`. It makes sense that if the application is accepted or rejected, it also means that the interview has been completed.
-Hence the `ApplicationStatus` will trigger `InterviewStatus` if the user is changing the `ApplicationStatus` from `Pending` to `Accepted` or `Rejected`.
-The `trigger mechanism` (WIP) is currently handled by `EditCommand#triggerInterviewStatus()`.
+This `EditPersonDescriptor` class will take in the current attributes of the Candidate, and update the fields accordingly by the
+`EditCommandParser`. 
 
 #### Why is the feature implemented as such?
 
-**1. Triggers for `ApplicationStatus` and `InterviewStatus` <br>
-For the `ApplicationStatus` and `InterviewStatus`, we initially thought of allowing the user to manually update individual statuses.
-(Eg. Updating `ApplicationStatus` to `Accepted` will not trigger `InterviewStatus`).
-We initially thought that this would be a cleaner way, and to also make sure there is no wrong information in the system.
-In the end, we went ahead with the triggers as it would make sense for the user to have all the statuses updated automatically,
-and we just need to make more checks in our code.
-The downside to this implementation is that the user will not be able to individually allocate the `InterviewStatus`, but we believe this
-will not be an use case for any user.
+**1. We have implemented the `EditCommand` in this manner because it will help enhance the user experience. 
+One alternative method we had was to allow the user to update the candidate by retyping all the fields of the Candidate. 
+However, this proved to be not efficient, and hence we implemented it this way. With `EditCommandParser` and `EditPersonDescriptor`, 
+the user can edit one attribute of the candidate without typing every attribute of the Candidate.  
 
 #### UML Diagram
-The following activity diagram summarizes what happens when a user executes a `edit` command for `ApplicationStatus` and `InterviewStatus` <br>
-<img src="images/StatusActivityDiagram.png" width="250" />
+The following activity diagram summarizes what happens when a user executes a `edit` command. <br>
+<img src="images/EditCommandDiagram.png" width="250" />
```

Comment:
Might want to consider increasing the width to 550! Looks quite small now at 250!

Reply:
ok sure will do !!